nitransforms/io/fsl.py

```diff
@@ -55,6 +69,35 @@ def from_string(cls, string):
         )
         return tf
 
+    def to_ras(self, moving=None, reference=None):
+        """Return a nitransforms internal RAS+ matrix."""
+        if moving is None:
+            warnings.warn(
+                "Converting FSL to RAS: moving image not provided, using reference."
+            )
+            moving = reference
+
+        if reference is None:
+            raise ValueError("Cannot build FSL linear transform without a reference")
+
+        reference = _ensure_image(reference)
+        moving = _ensure_image(moving)
+
+        refswp, refspc = _fsl_aff_adapt(reference)
+        pre = reference.affine.dot(np.linalg.inv(refspc).dot(np.linalg.inv(refswp)))
+
+        # Adjust for moving image offset and orientation
+        movswp, movspc = _fsl_aff_adapt(moving)
+        post = np.linalg.inv(movswp).dot(movspc.dot(np.linalg.inv(moving.affine)))
+
+        mat = self.structarr["parameters"].T
+
+        return (
+            np.linalg.inv(post)
+            @ np.swapaxes(np.linalg.inv(mat), 0, 1)
+            @ np.linalg.inv(pre)
```

Comment:
Seems to be a log of inversion.

```suggestion
        pre = refswp @ refspc @ np.linalg.inv(reference.affine)

        # Adjust for moving image offset and orientation
        movswp, movspc = _fsl_aff_adapt(moving)
        post = moving.affine @ movspc @ np.linalg.inv(movswp)

        mat = self.structarr["parameters"].T

        return post @ np.swapaxes(np.linalg.inv(mat), 0, 1) @ pre
```

I'll note that pre- and post- are not fully symmetric, as I would have expected `movspc` to be inverted the same as `movswp`. If that's a bug:

```suggestion
        pre = refswp @ refspc @ np.linalg.inv(reference.affine)

        # Adjust for moving image offset and orientation
        movswp, movspc = _fsl_aff_adapt(moving)
        post = moving.affine @ np.linalg.inv(movspc) @ np.linalg.inv(movswp)

        mat = self.structarr["parameters"].T

        return post @ np.swapaxes(np.linalg.inv(mat), 0, 1) @ pre
```

Just another note: with this much use, I'd probably just `from numpy.linalg import inv` instead of specifying it fully every time.